src/components/BackgroundGrid/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import classes from './index.module.scss'
+
+export const BackgroundGrid: React.FC = () => {
+  return (
+    <div className={[classes.backgroundGrid, 'grid'].filter(Boolean).join(' ')}>
+      <div className={[classes.column, 'cols-4'].filter(Boolean).join(' ')}></div>
+      <div className={[classes.column, 'cols-4'].filter(Boolean).join(' ')}></div>
+      <div className={[classes.column, 'cols-4'].filter(Boolean).join(' ')}></div>
+      <div className={[classes.column, 'cols-4'].filter(Boolean).join(' ')}></div>
```

Comment:
@paulpopus Might need you to explain to me in a huddle of what's going on here. Maybe I'm just missing something here but removing the last (4th) `div` element here doesn't seem to be affecting anything

Is the 4th div element just overlapping something else and is not needed? Not sure